UPGRADE.md

```diff
@@ -8,6 +8,13 @@ awareness about deprecated code.
 
 # Upgrade to 3.3
 
+## Deprecated `Connection::getWrappedConnection()`, `Connection::connect()` made `@internal`.
+
+The wrapper-level `Connection::getWrappedConnection()` method has been deprecated.
+Use `Connection::getNativeConnection()` to access the native connection.
+
+The `Connection::connect()` method has been marked internal. It will be marked `protected` in DBAL 4.0.
```

Comment:
Should we use the FQCN here, to make clear we're talking about the `Connection` class and not the `Driver\Connection` interface?

Reply:
The first sentence explicitly says "wrapper-level", so there's no ambiguity. The second one doesn't change the context, so it also implies the wrapper-level connection. Additionally, there's no `connect()` in `Driver\Connection`.